packages/pv-create-component/.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
```

Comment:
what is this travis task doing?

Reply:
was meant for the case when this package is not part of the fe-tools mono repo anymore. I removed this file for the time being.